### PR TITLE
Host Config and CSS for Viva Connections - Dark theme edition

### PR DESCRIPTION
**Pull request by lynamemi**
# Description

This PR introduces a host config and css that support dark theme for Viva Connections in the Adaptive Card Designer.

[Here](https://github.com/microsoft/AdaptiveCards/pull/6598) is the original PR that introduced Viva Connections by @SidGulatiMsft 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/7164)

### CODE REVIEW

File: source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ export const defaultMicrosoftHosts: containers.HostContainer[] = [
 	new containers.OutlookContainer("Outlook Actionable Messages", "containers/outlook-container.css"),
 	new containers.DarkTeamsContainer("Microsoft Teams - Dark", "containers/teams-container-dark.css"),
 	new containers.LightTeamsContainer("Microsoft Teams - Light", "containers/teams-container-light.css"),
-	new containers.VivaConnectionsContainer("Viva Connections", "containers/viva-connections-container.css"),
+	new containers.VivaConnectionsContainerDark("Viva Connections - Dark", "containers/viva-connections-container-dark.css"),
```
Comment (licanhua) [inline source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts:14]: I would suggest name `DarkVivaConnectionsContainer` and LightVivaConnectionsContainer` to match with name format like DarkTeamsContainer

**Review by jwoo-msft — APPROVED** [review]

File: source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts [hunk 2]
```diff
@@ -11,7 +11,8 @@ export const defaultMicrosoftHosts: containers.HostContainer[] = [
 	new containers.OutlookContainer("Outlook Actionable Messages", "containers/outlook-container.css"),
 	new containers.DarkTeamsContainer("Microsoft Teams - Dark", "containers/teams-container-dark.css"),
 	new containers.LightTeamsContainer("Microsoft Teams - Light", "containers/teams-container-light.css"),
-	new containers.VivaConnectionsContainer("Viva Connections", "containers/viva-connections-container.css"),
+	new containers.DarkVivaConnectionsContainer("Viva Connections - Dark", "containers/viva-connections-container-dark.css"),
```
Comment (SidGulatiMsft) [inline source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts:14]: Maybe "Viva Connections - Dark **(Preview)**"??

File: samples/HostConfig/viva-connections-light.json [hunk 3]
```diff
@@ -67,8 +67,8 @@
 					"subtle": "#E1DFDD"
 				},
 				"accent": {
-					"default": "#6264A7",
-					"subtle": "#A6A7DC"
+					"default": "#5B5FC7",
+					"subtle": "#C5CBFA"
```
Comment (SidGulatiMsft) [inline samples/HostConfig/viva-connections-light.json:71]: Why do we have to make changes in this file??
Intuitively, light theme shouldn't be altered by the introduction of a dark theme.

File: source/nodejs/adaptivecards-designer/src/containers/viva-connections/viva-connections-container-light.css [hunk 4]
```diff
@@ -25,9 +25,9 @@
   padding: 5px 8px;
   bottom: 12px;
   overflow: hidden;
-  border-color: #696baa;
+  border-color: #7579EB;
   background-color: #ffffff;
-  color: #6264a7;
```
Comment (SidGulatiMsft) [inline source/nodejs/adaptivecards-designer/src/containers/viva-connections/viva-connections-container-light.css:30]: Same.
Why are we making changes in the light.css file??

**Review by SidGulatiMsft — APPROVED** [review]

File: samples/HostConfig/viva-connections-light.json (hunk 3 above)
Comment (lynamemi) [reply samples/HostConfig/viva-connections-light.json:71]: Great question - When I went in search of the Teams dark theme palette, I discovered that there are new primary colors for the light palette as well. Updated them in this PR to be consistent.

File: source/nodejs/adaptivecards-designer/src/containers/viva-connections/viva-connections-container-light.css (hunk 4 above)
Comment (lynamemi) [reply source/nodejs/adaptivecards-designer/src/containers/viva-connections/viva-connections-container-light.css:30]: See other reply!

**Pull request merged**